tests/python/unittest/test_pass_bound_checkers.py

```diff
@@ -37,6 +37,7 @@ def lower(sch, args):
     bounds = tvm.schedule.InferBound(sch)
     stmt = tvm.schedule.ScheduleOps(sch, bounds)
     stmt = tvm.ir_pass.LoopPartition(stmt, True)
+    stmt = tvm.ir_pass.RemoveNoOp(stmt)
```

Comment:
Ideally, we should put the call to RemoveNoOp in the loop partition pass itself to get the cleaner output. See #3726.  But it breaks some tests in the `test_pass_loop_partition`. So for now, just fixing it here.